model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/resource_utilization_data.py

```diff
@@ -238,7 +285,7 @@ def requires_mixed_precision(in_model: Any,
     total_weights_memory_bytes = 0 if len(weights_memory_by_layer_bytes) == 0 else sum(weights_memory_by_layer_bytes)
 
     # Compute max activation tensor in bytes
-    activation_output_sizes_bytes, _ = compute_activation_output_sizes(transformed_graph)
+    activation_output_sizes_bytes, _ = compute_activation_output_maxcut_sizes(transformed_graph)
```

Comment:
change the variable name (maybe to generic "activation_memory_estimation_bytes" or something like that)